src/auxiliary/t8code.jl

```diff
@@ -32,10 +32,12 @@ function init_t8code()
             T8code.Libt8.p4est_init(C_NULL, LOG_LEVEL)
         end
 
+        # Clean up t8code before MPI shuts down.
         MPI.add_finalize_hook!() do
+            T8code.clean_up()
```

Comment:
Codecov reports this line to be uncovered. I have the feeling it would be important to cover this. Can you add a reasonable test?

Reply:
Thanks for nerd sniping me again ... I added tests.